### PR TITLE
Update to Tool Management slides for Admin Training 2021

### PR DESCRIPTION
**Pull request by cat-bro**
Some proposed updates to the toolshed/tool management slides:

Additions:
- Three slides at the beginning talking about the tool panel and tool configuration files
- Slide containing a screenshot of the toolshed
- `shed-tools install` example slide
- two slides on `shed-tools test`

Changes:
- `workflow-to-tools` slide instead of `workflow-install`

Deletions:
- `run-data-managers` slide
- Dependency resolvers example
- Slide about integrated tool panel (this is covered in added slides on tool configuration files)

Some slides have been rearranged as well.  The approximate running time is 20 minutes.

For things that I have taken out, I can easily put these back if they are missed.

### CODE REVIEW

File: topics/admin/tutorials/tool-management/slides.html [hunk 1]
```diff
@@ -351,55 +694,32 @@
...
 - You can read more about this in the [Reference Genomes in Galaxy slides](../reference-genomes/slides.html)
-
----
-
-## Suite repositories
-
-- Suites can be used to split up a set of tools into multiple repositories
-  - A suite is a single repository that 'depends' on many others
-  - When you install the suite, all 'dependency repositories' will be installed too
-- With a tweak to `.shed.yml`, [Planemo](https://planemo.readthedocs.io) can upload a directory of tools to the TS as a suite of separate tools
```
Comment (nsoranzo) [inline topics/admin/tutorials/tool-management/slides.html:370]: Not sure where to put it, but I'd keep the slide on Suites.

Comment (cat-bro) [reply topics/admin/tutorials/tool-management/slides.html:370]: I'll put it back where it was (after the slide on reference data).  It fits into the 'More on Tool Management' slides

**Comment by cat-bro** [comment]
Thank you @nsoranzo!

File: topics/admin/tutorials/tool-management/slides.html [hunk 2]
```diff
@@ -723,6 +723,25 @@
...
+
+- Suites can be used to split up a set of tools into multiple repositories
+  - A suite is a single repository that 'depends' on many others
+  - When you install the suite, all 'dependency repositories' will be installed too
+- With a tweak to `.shed.yml`, [Planemo](https://planemo.readthedocs.io) can upload a directory of tools to the TS as a suite of separate tools
+- Examples: suite_samtools, suite_hicexplorer
+
+???
+
+There are also repositories in the Tool Shed that are suite repositories.  These can be used
```
Comment (hexylena) [inline topics/admin/tutorials/tool-management/slides.html:736]: Would you mind making each sentence a bullet point? Otherwise this looks ok so far.

Comment (cat-bro) [reply topics/admin/tutorials/tool-management/slides.html]: Thanks @hexylena. I've updated the speaker notes (there were quite a few typos) and split up the sentences to make it more readable.

Comment (hexylena) [reply topics/admin/tutorials/tool-management/slides.html]: awesome!

**Review by hexylena — APPROVED** [review]
I like this, it looks good!

**Comment by hexylena** [comment]
Thanks so much @cat-bro. 

I'll rebase get the automated video going over in #2152.

**Pull request merged**